src/test/java/com/sigma/cinematicketpro/service/impl/MovieServiceImplTest.java

```diff
@@ -0,0 +1,138 @@
+package com.sigma.cinematicketpro.service.impl;
+
+import com.sigma.cinematicketpro.TestUtils;
+import com.sigma.cinematicketpro.entity.Movie;
+import com.sigma.cinematicketpro.exception.ResourceNotFoundException;
+import com.sigma.cinematicketpro.repository.MovieRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class MovieServiceImplTest {
+
+    @Mock
+    private MovieRepository movieRepository;
+    @InjectMocks
+    private MovieServiceImpl sut;
+
+    private static final List<Movie> movieList = TestUtils.getValidMovies();
```

Comment:
really you dont need a list in majority of th cases only one movie

Reply:
fixed